pool/src/lib.rs

```diff
@@ -767,6 +768,58 @@ impl LendingPool {
 
         Ok(())
     }
+
+    /// Calculate interest rate IR = MIN [ max_rate, base_rate / (1 - U)^alpha]
+    /// where
+    /// U - utilization, U = total_debt / total_collateral
+    /// alpha - parameter, by default 1.43 expressed as 143 with denominator 100
+    /// max_rate - maximal value of interest rate, by default 500% expressed as 50000
+    /// base_rate - base interest rate, by default 2%, expressed as 200
+    ///
+    /// For (1-U)^alpha calculation use binomial approximation with three terms
+    /// (1-U)^a = 1 - alpha * U + alpha/2 * (alpha - 1) * U^2 - alpha/6 * (alpha-1) * (alpha-2) * U^3
+    #[allow(dead_code)]
+    fn calc_interest_rate(
```

Comment:
As we discussed on the meeting, let's also add some tests to verify it works as expected under usual and unusual circumstances.